public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -313,7 +313,7 @@ export class PanelModel implements DataConfigSource, IPanelModel {
       timezone: dashboardTimezone,
       timeRange: timeData.timeRange,
       timeInfo: timeData.timeInfo,
-      maxDataPoints: this.maxDataPoints || width,
+      maxDataPoints: this.maxDataPoints || Math.trunc(width),
```

Comment:
[Math.trunc](https://caniuse.com/?search=math.trunc) doesn't seem to be supported by InternetExplorer. Could it be a problem? Do we have polyfill for it?

Reply:
good catch @ifrost ... i replaced it with `Math.floor`, it should be good enough considering `width` is never negative.